src/components/EmptyState/emptyState.scss

```diff
@@ -32,7 +32,7 @@
         margin-bottom: 20px;
     }
     .cta {
-        height: 36px;
+        height: 40px;
```

Comment:
Is it intentional?

Reply:
![image](https://user-images.githubusercontent.com/98738644/173044252-37041ba1-2d48-46ef-b762-b99b3d7ce46b.png)

Reply:
css having same name creating issues in some elements

Reply:
Try to use the flexbox class to fix it instead of changing the height.

Reply:
done